rclpy/rclpy/topic_or_service_is_hidden.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+HIDDEN_TOPIC_PREFIX = '_'
+
+
+def topic_or_service_is_hidden(name):
```

Comment:
We should probably update existing code in `ros2cli` to use this new function once merged.